SPECS/nginx/CVE-2024-7347.1.patch

```diff
@@ -0,0 +1,48 @@
+From 7362d01658b61184108c21278443910da68f93b4 Mon Sep 17 00:00:00 2001
```

Comment:
How about we club both of these hunks in a single file:
```bash
cat CVE-2024-7347* > CVE-2024-7347.patch
```

Reply:
+1 to @mrfw.

Reply:
Done!